src/interpreter/runner.ts

```diff
@@ -3,7 +3,7 @@ import { BinaryExpression, Expression, isBinaryExpression, isBooleanExpression,
 import { createLoxServices } from "../language-server/lox-module";
 import { v4 } from 'uuid';
 import { URI } from "vscode-uri";
-import { CancellationToken } from "vscode-languageclient";
+import { CancellationToken } from 'vscode-jsonrpc';
```

Comment:
Was necessary to import less from vscode, otherwise the full initialization breaks unit test